.github/workflows/ci.yml

```diff
@@ -23,20 +19,6 @@ jobs:
         os: [ubuntu-18.04, windows-2019]
         php: ['7.3', '7.4', '8.0']
 
-    services:
-      db:
-        image: mariadb:latest
-        env:
-          MYSQL_USER: root
-          MYSQL_PASSWORD: password
-          MYSQL_DATABASE: servidor_testing
-        ports: ['3306:3306']
-        options: >-
-          --health-cmd "mysqladmin ping"
-          --health-interval 5s
-          --health-timeout 2s
-          --health-retries 5
-
```

Comment:
You don't need a DB for tests?

Reply:
It's [supposedly preinstalled](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md#mysql) (MySQL rather than MariaDB, but 5.7 is probably good enough) so it's mostly to see if it works without.

Haven't checked the Windows env, but I'm not overly fussed if that doesn't work :wink:

Reply:
Ah, I didn't know this!